podcasts/End of Year/Stories/YearOverYearStory.swift

```diff
@@ -0,0 +1,211 @@
+import SwiftUI
+import PocketCastsServer
+import PocketCastsDataModel
+
+struct YearOverYearStory: ShareableStory {
+    var duration: TimeInterval = 5.seconds
+
+    let identifier: String = "year_over_year"
```

Comment:
@ashiagr I defined the identifier of this story as `year_over_year`. Open to suggestions if you have any other ideas. :)